go/engine/deprovision_test.go

```diff
@@ -224,7 +214,7 @@ func TestCurrentDeviceRevoked(t *testing.T) {
 	// secret store (if possible).
 	fu := NewFakeUserOrBust(tc.T, "dpr")
 	arg := MakeTestSignupEngineRunArg(fu)
-	arg.StoreSecret = libkb.HasSecretStore()
+	arg.StoreSecret = tc.G.SecretStore != nil // generally true
```

Comment:
Unfortunately, we lose test coverage of the path where `SecretStore` is nil, which I think we still need to support. Perhaps have a separate test for that case, and factor out the common code?